### PR TITLE
test: Initial commit for testing headers and paragraphs

### PR DESCRIPTION
**Pull request by TRIPLE-ADE**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
+      <h1>Testing Header One</h1>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestias sed,
+        accusantium dolorum a est distinctio non itaque ipsam excepturi harum
+        nihil iure? Eum perspiciatis, animi ut ipsa facilis fugiat vitae.
+      </p>
+    </section>
+
+    <section>
+      <h1>Testing Header Two</h1>
```
Comment (Succynice) [inline index.html]: @TRIPLE-ADE pls change this header to `h2` and below it, add a `h3` with its paragraph

thanks

**Comment by Chiemella** [comment]
Nice observation @Succynice bro..

File: index.html [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
-
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>test-run</title>
+  </head>
+
+  <body>
```
Comment (Chiemella) [inline index.html]: Nice one bro @TRIPLE-ADE

**Review by Chiemella — COMMENTED** [review]
Job well done.

File: index.html (hunk 2 above)
Comment (TRIPLE-ADE) [reply index.html]: should i remove the space there

Comment (Succynice) [reply index.html]: @TRIPLE-ADE it's not necessary boss

**Pull request closed without merging**